### DISCUSSION TITLE
How to launch the default browser by a click of a button?

### DISCUSSION THREAD

**Discussion by wschume** [flet-dev/flet#3321] [Q&A] [answered]
### Question

Is it possible to launch the default web browser with an URL from the click of a button?
Target platforms are primarily Android and iOS.

A typical use case could be:
Design an `IconButton` with flet as its icon, which launches the browser with the url: `https://flet.dev/` when clicked.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Use the [`url`](https://flet.dev/docs/controls/filledbutton#url) property of the Buttons.
Or call [`page.launch_url()`](https://flet.dev/docs/controls/page#launch_urlurl) in `on_click`.

**Reply by wschume** [reply]
I misread the documentation and thought that this would only work in web mode. Thank you for your help!
